.github/workflows/tests-ubuntu.yml

```diff
@@ -26,19 +26,19 @@ jobs:
         - python-version: pypy3
           env:
             TOXENV: pypy3
-            PYPY_VERSION: 3.6-v7.3.3
+            PYPY_VERSION: 3.7-v7.3.9
 
         # pinned deps
-        - python-version: 3.6.12
+        - python-version: 3.7.13
           env:
             TOXENV: pinned
-        - python-version: 3.6.12
+        - python-version: 3.7.13
           env:
             TOXENV: asyncio-pinned
         - python-version: pypy3
           env:
             TOXENV: pypy3-pinned
-            PYPY_VERSION: 3.6-v7.2.0
+            PYPY_VERSION: 3.7-v7.3.5
```

Comment:
Now that these run 3.7, the regular 3.7 job at the top of the list can be removed now.